.travis/awx_job_launch.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I don't see anything trigger this script. Could you explain how does it work?

Reply:
I'll remove this temporarily until we are ready to config auto launch.